src/sentry/integrations/gitlab/client.py

```diff
@@ -334,26 +334,14 @@ def get_file(self, repo: Repository, path: str, ref: str, codeowners: bool = Fal
         Path requires file path and ref
         file_path must also be URL encoded Ex. lib%2Fclass%2Erb
         """
-        from base64 import b64decode
 
         project_id = repo.config["project_id"]
         encoded_path = quote(path, safe="")
-        request_path = GitLabApiClientPath.file.format(project=project_id, path=encoded_path)
-        headers = (
-            {
-                "Accept": "application/vnd.github.raw",
-                "Content-Type": "application/raw; charset=utf-8",
-            }
-            if codeowners
-            else {}
-        )
-        contents = self.get(request_path, params={"ref": ref}, raw_response=True, headers=headers)
+        request_path = GitLabApiClientPath.file_raw.format(project=project_id, path=encoded_path)
```

Comment:
do you know if this `get_file` function is/can be used to get files other than codeowners? if it does, we may still want to use the `codeowners` flag in the parameters to only get the raw file if we're getting a codeowners file

Reply:
Well, the question is why wouldn't we want _all_ files as raw? Getting encoded file and returning `b64decode(contents["content"]).decode("utf-8")` is effectively the same as getting it raw and returning `contents.content.decode("utf-8")`, minus all the decoding work. 

My only doubt is re `.decode("utf-8")`, which doesn't make sense if it's a binary file like let's say an image. But in the old code it's happening regardless.

Reply:
that makes sense - thanks!